app/models/repaymentHistory/RepaymentHistoryUtils.scala

```diff
@@ -46,6 +47,14 @@ object RepaymentHistoryUtils {
     }
   }
 
+  private def getPoaChargeLinkUrl(isAgent: Boolean, taxYear: Int, chargeId: String): String = {
```

Comment:
next story created as tech debt feel free to comment: https://jira.tools.tax.service.gov.uk/browse/MISUV-8654